lib/matplotlib/legend.py

```diff
@@ -77,7 +77,7 @@ def finalize_offset(self):
         if self._update == "loc":
             self._update_loc(self.get_loc_in_canvas())
         elif self._update == "bbox":
-            self._bbox_to_anchor(self.get_loc_in_canvas())
+            self._update_bbox_to_anchor(self.get_loc_in_canvas())
```

Comment:
Does that mean none of this is tested?

Reply:
It only gets triggered when dragging a legend, afaik mpl isn't testing any interactivity, is it?

Reply:
I think it's possible to test interactivity.  However, whoever put the typo in didn't test, or tested the wrong thing, so perhaps add to the PR description  whatever test you are using to ensure this is the correct fix?

Reply:
It is possible to test the interactive code, but I think that is way too big of an ask to get a 1 line patch in!

Reply:
Yes, sorry, I wasn't meaning to ask for that if its hard, but at least some idea how this should be tested manually.

Reply:
@daniilS, since you seem to be doing some interactive PRs, you might be interested in taking a look here for how zoom and other interactive things like widgets are tested: https://github.com/matplotlib/matplotlib/blob/b636e9ed82b930ad4bde0b51360855fa79a7b438/lib/matplotlib/tests/test_backend_bases.py#LL142C3-L142C3

Reply:
@greglucas good point - I think all my other PRs were visual changes or toolbar behaviour which is hard to test automatically, but I might be able to write a test for dragging legends as the interaction is happening entirely within the axes.